### PR TITLE
add text.Handler.EnableArgs

### PR DESCRIPTION
**Pull request by achille-roussel**
@f2prateek 

Follow up of our earlier conversation, add a way to control the output of the text handler that doesn't depend on how the events were generated.

### CODE REVIEW

File: text/handler_test.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 func TestHandler(t *testing.T) {
 	b := &bytes.Buffer{}
 	h := NewHandler("==> ", b)
+	h.EnableArgs = true
```
Comment (f2prateek) [inline text/handler_test.go]: We should probably have a couple of tests to make bump coverage.

**Comment by f2prateek** [comment]
LGTM

File: text/handler_test.go (hunk 1 above)
Comment (achille-roussel) [inline text/handler_test.go]:
```
$ go test -cover ./text
ok  	github.com/segmentio/events/text	0.006s	coverage: 97.3% of statements
```
Is that not enough?

Comment (achille-roussel) [inline text/handler_test.go]: I guess you mean having one test with `EnableArgs = false`

Comment (f2prateek) [inline text/handler_test.go]: Yup

**Comment by achille-roussel** [comment]
Thanks for reviewing!

**Pull request merged**